ZenPacks/zenoss/ZenPackLib/lib/base/ComponentBase.py

```diff
@@ -231,6 +231,13 @@ def get_containing_relname(self):
         """Return name of containing relationship."""
         for relname, relschema in self._relations:
             if issubclass(relschema.remoteType, ToManyCont):
+                # skip if it's not the real containing rel
+                # for instance an inherited os.filesystems for sub-classed FileSystem
+                rel = getattr(self, relname, None)
```

Comment:
Yes that's correct in one sense (they are invalid/non-functioning) but are a side effect of using the proxy classes.  So far it has only surfaced here.  One potential alternative fix would be for ZPL to defer to ZenModel's containing relations, but I bet that would have all kinds of implications.